index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/1ca239d8ab.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Version Portfolio</title>
+</head>
+<body>
+  <div class="container">
+
+    <section class="container-section">
+    <header>
+      <nav class="nav-header">
+        <div class="logo">My Logo</div>
+        <span class="bars"><i class="fa-solid fa-bars"></i></span>
+      </nav>
+    </header>
+    
+      <div class="intro">
+        <h1>I'm Dinma <br> Glad to see <br> you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        
+        <h4 class="connect">LET'S CONNECT</h4>
+        
+        <ul class="socials">
+          <li>
+            <a href="#"><img src="./img/angellist.svg" alt="Angellist-Icon"></a>
+          </li>
+          <li><a href="#">
+            <img src="./img/github.svg" alt="Githud-Icon"></a>
+          </li>
+          <li><a href="#">
+            <img src="./img/linkedin-icon.svg" alt="LinkedIn-Icon"></a>
+          </li>
+          <li><a href="#">
+            <img src="./img/twitter.svg" alt="Twitter-Icon"></a>
+          </li>
+          <li><a href="#">
+            <img src="./img/medium.svg" alt="Medium-Icon"></a>
+          </li>
+        </ul>
```

Comment:
- [ ] Great job placing your icons in a `ul` tag. Kindly place the icons in the right order as in the template 😄 
- Your site:
![Annotation 2022-03-15 171215](https://user-images.githubusercontent.com/85113567/158425209-dca790ca-6e94-45e8-b6c3-8853a72d0257.jpg)
- Figma design:
![Annotation 2022-03-15 171201](https://user-images.githubusercontent.com/85113567/158425242-a44b233e-6695-45de-b060-ad3438835681.jpg)

Reply:
Hello @Kingstalux, as requested by you, I have : 
- Removed the div of class logo from the nav tag and wrapped the logo in an <a> tag
- Changed the <div = class intro> to <section>
- Applied the font-weight to h1 element
- Placed the icons in the right order as in the template